CHANGELOG.next.asciidoc

```diff
@@ -29,6 +29,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Remove Journalbeat. Use `journald` input of Filebeat instead. {pull}29131[29131]
 - `include_matches` option of `journald` input no longer accepts a list of string. {pull}29294[29294]
 - Add job.name in pods controlled by Jobs {pull}28954[28954]
+- Fix annotations enrichment. {pull}29605[29605]
```

Comment:
This is not very descriptive of your change. Something like `Enrich kubernetes metadata with node annotations`